### DISCUSSION TITLE
When exporting to PDF, Korean is not visible.

### DISCUSSION THREAD

**Discussion by sunny0678** [GitbookIO/community#409] [5. Export]
### Ask a question, start a conversation, or make an annoucement

Currently using Gitbook Pro license. 

When exporting to PDF, Korean is not visible. Isn't this a problem that occurs because the Korean font does not exist when exporting?
Our company became an annual member of PRO in order to export Korean documents to PDF. Failure to support this will have serious implications for work.

**Comment by aikoii** [comment]
@sunny0678 I'm sorry for those troubles and how it affected your work! I'm afraid our PDF at the moment doesn't support the Korean language and some blocks in the editor. Have you thought about using [shareable links](https://docs.gitbook.com/published-documentation/share/share-links#publish-with-share-links) instead? If it doesn't work for you, I'm happy to provide a refund 🙏.

**Comment by jungparc** [comment]
@aikoii Hi, is there any chance to fix this issue? The Export feature would greatly help us share the content with our customers.

**Reply by sunny0678** [reply]
It‘s fixed issue.
